README.md

```diff
@@ -93,6 +93,30 @@ First, update the repository (assuming you have `master` checked out):
 Look carefully for errors with `git pull`, and reset locally modified files if
 necessary.
 
+## Upgrading Instructions 
+
+*__Only necessary while upgrading from v1.3.2 or below.__*
+
+1. Drop the old dcrdata database and create a new empty dcrdata database.
+ ```sql
+ // drop the old database
+ DROP DATABASE dcrdata;
+
+// create a new database with the same user set in the dcrdata.conf e.g user dcrdata
+CREATE DATABASE dcrdata WITH OWNER dcrdata;
+
+// grant all permissions to user dcrdata
+GRANT ALL PRIVILEGES ON DATABASE dcrdata to dcrdata;
+ ```
+
+2. Delete the data folder in dcrdata folder where dcrdata.conf file is located
+  - Mac :  `~/Library/Application Support/Dcrdata/data`
+  - Linux :  `~/Dcrdata/data`
+  - Window :  `C:\Users\<your-username>\AppData\Local\Dcrdata\data` 
+  
+3. Restart the data migration again by running the dcrdata and dcrd binaries
+  - run the downloaded dcrd binary first then the generated dcrdata binary
+
```

Comment:
@chappjc I added this Upgrading instructions to address the last issue you just raised.

Reply:
I missed that it contained the `DROP DATABASE dcrdata` instruction so that's good.  In addition to this, it should be clear enough.

https://github.com/decred/dcrdata/pull/427/files#diff-60206eba7205ea27cd7a8137d424f61cR269

It's unfortunately there's no possibility for upgrade, only rebuild, so we'll need to put in a few PRs ahead of this.  But it's in good shape I think.

Reply:
The link in the above post redirects to this page. Was that the intention?